src/game/PowerGrid.hxx

```diff
@@ -0,0 +1,23 @@
+#ifndef POWERGRID_HXX_
+#define POWERGRID_HXX_
+
+#include "point.hxx"
+#include <vector>
+
+struct PowerPlant
+{
+  Point coordinate;
+  int powerProduction = 0;
+};
+
+class PowerGrid
+{
+public:
+  PowerGrid();
+
+private:
+  std::vector<PowerPlant> m_powerPlants;
+  std::vector<Point> doesnotwork;
```

Comment:
what is this doesnotwork ? can you change it to better naming ?

Reply:
No, this is a draft. It’s for me so I can keep better track. Please do not review that before I request a review on this. It’s like 20% done.